### PR TITLE
Optimobile

### PR DESCRIPTION
**Pull request by BBHOptimove**

### CODE REVIEW

File: NotificationService/NotificationService.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
```
Comment (optimove-andy) [inline NotificationService/NotificationService.swift:1]: This looks like its duped with `OptimoveNotificationService`

File: KumulosSdkSwift.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (optimove-andy) [inline KumulosSdkSwift.podspec]: Some Kumulos references to delete.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Changelog
```
Comment (optimove-andy) [inline CHANGELOG.md:2]: Will 3.6.0 be released or will we replace this changelog / release version 4.0.0 directly.

File: NotificationService/NotificationService.swift (hunk 1 above)
Comment (optimove-andy) [reply NotificationService/NotificationService.swift:1]: Suggest we just roll with `NotificationService` you already have the Optimove SDK, we probably dont need to qualify it again?

File: Optimobile/AppDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+//
+
+import UIKit
+import KumulosSDK
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let builder = KSConfigBuilder(apiKey: "2b16f7c7-6f0e-46c1-979f-1c13b5fd8a0f", secretKey: "Xh62vshcS15xJCjSHiQ2dcPvfXHVirKdT+Vg").enableInAppMessaging(inAppConsentStrategy: InAppConsentStrategy.AutoEnroll).setPushOpenedHandler(pushOpenedHandlerBlock: { (notification : KSPushNotification) -> Void in
```
Comment (optimove-andy) [inline Optimobile/AppDelegate.swift:15]: Is there a test app as part of the SDK bundle? Im not used to seeing credentials checked in.

File: Optimobile/AppDelegate.swift [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            let deepLink = buttonPress.deepLinkData
+            let messageData = buttonPress.messageData
+            print(deepLink)
+            print(messageData ?? "")
+            // TODO: Inspect the deep link & message data to perform relevant action
+        }).setPushOpenedHandler(pushOpenedHandlerBlock: { (notification : KSPushNotification) -> Void in
+            print(notification)
+        }).enableDeepLinking({ (resolution) in
+            print("Deep link resolution result: \(resolution)")
+        }).enableCrash()
```
Comment (optimove-andy) [inline Optimobile/AppDelegate.swift:34]: Crash features should be stripped out, is plan to retain the KSConfig builder approach or to call methods on the singleton?

**Comment by BBHOptimove** [comment]
fixed by andy׳s comments on prev pr

**Comment by optimove-andy** [comment]
Closing as we are merging OptiMobile -> dev with further changes.

**Pull request closed without merging**